sources/packages/api/src/database/entities/education-program.model.ts

```diff
@@ -157,7 +158,7 @@ export class EducationProgram extends RecordDataModel {
   @Column({
     name: "approval_status",
   })
-  approvalStatus: string;
+  approvalStatus: ApprovalStatus;
```

Comment:
Nice work that we have used enum for Approval Status. Do we have a plan or backlog ticket to change the database column to use enum respectively?

Reply:
added in story, https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/bcgov/sims/578